test/scenarios.py

```diff
@@ -3517,6 +3517,7 @@
         {
             "DROP_PEER_REN": ActionList(
                 [
+                    action.ConfigureNetwork(latency="200ms"),
```

Comment:
Why so much latency here?

Reply:
Actually no reason for it to be there as testrunner queues the events, removed this one :)